symfony/plugins/orangehrmPimPlugin/config/doctrine/schema.yml

```diff
@@ -486,14 +486,16 @@ EmpDirectdebit:
 EmployeeMembership:
   tableName: hs_hr_emp_member_detail
   columns:
+    id:
+      type: integer(4)
```

Comment:
why is the "id" column int(6) in the sql query but int(4) in the doctrine schema?